packages/editor-ui/src/components/NodeExecutionErrorMessage.vue

```diff
@@ -1,16 +1,26 @@
 <script lang="ts" setup>
+/* eslint-disable vue/no-v-html */
 import { useI18n } from '@/composables/useI18n';
+import { sanitizeHtml } from '@/utils/htmlUtils';
+
+import { computed } from 'vue';
+
 const i18n = useI18n();
 
-defineProps<{
+const props = defineProps<{
 	nodeName: string;
 	errorMessage?: string;
 }>();
+
+const sanitizedErrorMessage = computed(() => {
+	return props.errorMessage ? sanitizeHtml(props.errorMessage) : '';
+});
 </script>
 
 <template>
 	<div>
-		{{ errorMessage }}<br />
+		<span data-test-id="sanitized-error-message" v-html="sanitizedErrorMessage"></span>
```

Comment:
Hmm I thought we had a lint rule for this, but apparently not. We have `v-n8n-html` that sanitizes the html. We should preferably use that